resources/lang/ru_RU.yml

```diff
@@ -0,0 +1,271 @@
+CHANGE_VISIBILITY_STEP: "Выбрать видимость"
+CHANGE_VISIBILITY_EXPLICATION: "§n§7Public :§r\n  The faction will be free to all and everyone can come and go as they please\n \n§n§7Private :§r\n  The faction will be closed to invitations and players will have to be invited by a member\n \n§n§7By invitation :§r\n  Players will be able to send invitations to the faction and the faction will be able to invite players"
+MANAGE_FACTION_MAIN_TITLE: "Управление фракцией"
+MANAGE_PERMISSION_MAIN_TITLE: "Управление разрешениями"
+MEMBERS_LIST_TITLE: "Список участников"
+MANAGE_MEMBER_REQUEST_TITLE: "Управление запросом - {{name}}"
+REQUEST_TITLE: "Управление запросом - {{name}}"
+MANAGE_MEMBER_INVITATION_TITLE: "Управление приглашением - {{name}}"
+INVITATION_TITLE: "Управление приглашением - {{name}}"
+MANAGE_MEMBERS_REQUEST_LIST_TITLE: "Управление участниками - Запрос"
+REQUEST_LIST_TITLE: "Управление запросом"
+MANAGE_MEMBERS_INVITATION_LIST_TITLE: "Управление участниками - Приглашение"
+INVITATION_LIST_TITLE: "Управление приглашением"
+SEND_INVITATION_PANEL_TITLE: "Отправить приглашение"
+SEND_INVITATION_PANEL_CONTENT: "Чтобы вернуться назад, введите пустое поле"
+SEND_INVITATION_PANEL_INPUT_CONTENT: "Имя игрока :"
+SEND_INVITATION_PANEL_INPUT_CONTENT_FACTION: "Имя фракции :"
+MANAGE_MEMBER_PANEL_TITLE: "Управление - {{playerName}}"
+MANAGE_MEMBERS_MAIN_PANEL_TITLE: "Управление участниками"
+MANAGE_MEMBERS_LIST_PANEL_TITLE: "Управление участниками - Список"
+MEMBER_CHANGE_RANK_PANEL_STEP: "Выберите ранг"
+MEMBER_CHANGE_RANK_PANEL_TITLE: "Выберите ранг - {{playerName}}"
+CREATE_FACTION_PANEL_TITLE: "Создать фракцию"
+CREATE_FACTION_PANEL_INPUT_CONTENT: "Имя фракции : "
+CREATE_FACTION_PANEL_CONTENT: "Пожалуйста, введите название вашей фракции ниже, Чтобы вернуться назад, введите пустое поле"
+CREATE_FACTION_PANEL_CONTENT_BANNED: " §c>> §4Это название фракции запрещено"
+HOME_FACTION_PANEL_TITLE: "Список домов фракции"
+HOME_FACTION_PANEL_CONTENT: "§7Нажмите на нужный дом, чтобы телепортироваться туда"
+JOIN_FACTION_PANEL: "Присоединиться к фракции"
+MAIN_PANEL_TITLE_NO_FACTION: "Главное меню"
+MAIN_PANEL_TITLE_HAVE_FACTION: "Главное меню - {{factionName}}"
+TOP_FACTION_TITLE: "Топ фракции"
+TOP_FACTION_LINE: "§r §6> §7[§e{{rank}}§7]§f {{factionName}}  §o§7Уровень {{level}} \n"
+LEADERBOARD_TITLE: "Таблица лидеров"
+
+CONFIRMATION_TITLE_DELETE_FACTION: "Подтверждение удаления - {{factionName}}"
+CONFIRMATION_CONTENT_DELETE_FACTION: "§fВы уверены, что хотите удалить свою фракцию? Ты не сможешь вернуться"
+CONFIRMATION_TITLE_LEAVE_FACTION: "Подтверждение выхода - {{factionName}}"
+CONFIRMATION_CONTENT_LEAVE_FACTION: "Вы уверены, что хотите покинуть эту фракцию? Ты не сможешь вернуться"
+CONFIRMATION_TITLE_TRANSFER_PROPERTY: "Подтверждение передачи права собственности"
+CONFIRMATION_CONTENT_TRANSFER_PROPERTY: "§fВы уверены, что хотите передать право собственности на эту фракцию? Вы не сможете вернуться, и вас назначат Co-Leader"
```

Comment:
No equivalent to "Co-Leader", "Recruit" etc  for Russian?

Reply:
Ru have